easybuild/easyconfigs/p/PyMOL/PyMOL-2.4.1_198-foss-2019b.eb

```
@@ -0,0 +1,36 @@
+# #
+# This is a contribution from HPCNow! (http://hpcnow.com)
+# Copyright::   HPCNow!
+# Authors::     Jordi Blasco <jordi.blasco@hpcnow.com>
+# License::     GPL-v3.0
+# #
+
+easyblock = 'Tarball'
+
+name = 'PyMOL'
+version = '2.4.1_198'
+subversion = '-Linux-x86_64-py37'
+pythonver = '3.7.4'
```

These need to have a prefix to their variable names:
```suggestion
local_subversion = '-Linux-x86_64-py37'
local_pythonver = '3.7.4'
```

---

At second thought, perhaps `local_subversion` should just be a `versionsuffix` instead?